source/data/tier1data/features/collaboration.rst

```diff
@@ -0,0 +1,192 @@
+.. _collaboration: 
+
+##############################
+Sharing data and collaboration
+##############################
+
+Depending on your research project, different people might need different access to the data stored in Tier-1 Data.
+
+Some projects are very open: everyone can work on all data in the project.  
+Other projects, especially those with sensitive data, might need to be more strict in who can access which data and in which manner.
+Tier-1 Data allows you to manage this in detail by assigning different levels of access to *groups*, which can consist of any number of users.
+
+****************
+Groups and users
+****************
+In Tier-1 Data, individual users can be put together in groups. 
+Currently, we distinguish between two groups: project groups and access groups.  
+
+A **project group** is created when a research group applies for a project in Tier-1 Data.
+It includes all collaborators on the project, some of which are **responsibles**, and has a shared collection.
```

Comment:
should the 'responsibles' be 'managers'? Or at least we should talk about roles?

Reply:
Will be explained in the group management page, which I will add next time. I'll remove the mention of responsibles for now.